docs/wfe.rst

```diff
@@ -0,0 +1,216 @@
+Representing sources of wavefront error
+=======================================
+
+When used for Fraunhofer-domain propagation, POPPY supports expressing aberrations as if they were applied in a pupil plane. Currently, there is the `~poppy.ZernikeWFE` optical element to represent wavefront error as a combination of Zernike terms, and the `~poppy.ParameterizedWFE` optical element that offers the ability to specify basis sets other than Zernikes.
```

Comment:
Re: fresnel: noted.

Re: pupil vs image: that's currently how the ZernikeWFE (and WavefrontError) class works. It supplies a plane type at instantiation. That might be something to change...